Source/Shaders/BillboardCollectionVS.glsl

```diff
@@ -328,16 +328,18 @@ if (lengthSq < disableDepthTestDistance) {
     vec4 pEC1 = addScreenSpaceOffset(positionEC, dimensions, scale, vec2(0.0), origin, labelTranslate, pixelOffset, alignedAxis, validAlignedAxis, rotation, sizeInMeters, rotationMatrix, mpp);
     float globeDepth1 = getGlobeDepth(pEC1);
 
-    if (globeDepth1 != 0.0 && pEC1.z < globeDepth1)
+    float depthsilon = 10.0;
```

Comment:
Where did you come up with `10.0`? Have you tested other areas of terrain?

Great variable name.

Reply:
Yeah, I tested on both flat areas and mountainy areas.  10 was the lowest value that mostly got rid of the flickering.  Other values I tested like 5 and 7 still had a lot of flickering when the terrain had more drastic slopes